### PR TITLE
minor changes, renames

### PR DESCRIPTION
**Pull request by davids91**

### CODE REVIEW

File: assets/shaders/viewport_render.wgsl [hunk 1]
```diff
@@ -479,9 +479,9 @@ fn get_by_ray(ray_: Line) -> OctreeRayIntersection{
     var root_bounds = Cube(vec3(0.,0.,0.), f32(octreeMetaData.octree_size));
     let root_intersection = cube_intersect_ray(root_bounds, ray);
     if(root_intersection.hit){
-        current_d = impact_or(root_intersection, 0.);
+        ray_current_distance = impact_or(root_intersection, 0.);
```
Comment (nerdachse) [inline assets/shaders/viewport_render.wgsl:482]: Thatäs a good one! Perfect!

File: assets/shaders/viewport_render.wgsl [hunk 2]
```diff
@@ -292,7 +292,7 @@ fn get_node_occupancy_bitmap(sized_node_meta: u32) -> u32 {
 }
 
 // Functionality-wise this function is more generic, than its coutnerpart
```
Comment (nerdachse) [inline assets/shaders/viewport_render.wgsl:294]: Maybe we need tom top-level docs to explain what a brick is? I mean, there is so much jargon flying around... if you dive deep, you need to read papers, for sure, but generally, we could try to get some basic explanations in.

Is that something we want?

File: src/spatial/mod.rs [hunk 3]
```diff
@@ -14,21 +14,21 @@ pub(crate) const FLOAT_ERROR_TOLERANCE: f32 = 0.00001;
     derive(serde::Serialize, serde::Deserialize)
 )]
 pub(crate) struct Cube {
-    pub(crate) min_position: V3c<u32>,
-    pub(crate) size: u32,
+    pub(crate) min_position: V3c<f32>,
+    pub(crate) size: f32,
```
Comment (nerdachse) [inline src/spatial/mod.rs:18]: Huh, what motivated this change?

File: assets/shaders/viewport_render.wgsl (hunk 2 above)
Comment (davids91) [reply assets/shaders/viewport_render.wgsl:294]: Oh yes definitely, will include it in this PR

File: src/spatial/mod.rs (hunk 3 above)
Comment (davids91) [reply src/spatial/mod.rs:18]: Basically this flamegraph analysis: 
image
A mentionable amount of compute went into converting this back and forth.. In the end this change made the CPU raycast benchmarks 8-15% faster

Comment (nerdachse) [reply src/spatial/mod.rs:18]: Fascinating, I wonder if the same would happen with just a u32 or a f32... I mean, both are just 32 bits wide, and what those bits encode is up to interpretation...

File: assets/shaders/viewport_render.wgsl (hunk 2 above)
Comment (nerdachse) [inline assets/shaders/viewport_render.wgsl:294]: May I suggest a: `coutnerpart` -> `counterpart`. I am not sure How I can add a "suggested change"

Comment (davids91) [reply assets/shaders/viewport_render.wgsl:294]: Sure! :) It's the button on the top left by the way: 
image

File: src/spatial/mod.rs (hunk 3 above)
Comment (davids91) [reply src/spatial/mod.rs:18]: I'm not sure if the size match would guarantee that adequate conversion is lightweight though. 

On the same note: https://en.wikipedia.org/wiki/Fast_inverse_square_root 
Uses similar ideas!

**Pull request merged**